apps/06SpaceTravelToMars.js

```diff
@@ -9,7 +9,29 @@ var jobTypes = {
   programmer: 'Any Ship!'
 };
 
-// Your code here
+function CrewMember(name, job, specialSkill){
+  this.name = name;
+  this.job = job;
+  this.specialSkill = specialSkill;
+  this.ship = null;
+  this.enterShip = function(crewmember) {
+    this.crew.push(crewmember);
+  }
```

Comment:
is `this.crew` defined on a crew member?

Also, if you're going to write `crewmember`, write it as `crewMember`. [camelCase](https://en.wikipedia.org/wiki/CamelCase).

And, one last question. Does `enterShip()` take a `CrewMember`, or a `Ship`?